src/components/signature/signature.tsx

```diff
@@ -0,0 +1,44 @@
+import differenceInCalendarDays from 'date-fns/differenceInCalendarDays';
+import React from 'react';
+import styled from 'styled-components';
+
+export interface SignatureProps {
+  author: string;
+  date: string;
```

Comment:
If you make the date a string the component needs to know how to parse it. This makes it dependent of the context in which it gets used (and vice versa), so less reusable. 
I would propose to make the property of a type "Date". Then, the component doesn't need to take care of the format at all.

Reply:
Thanks for suggestion, I wanted to be consistens, as soon as we have date as a string in .md-files
I would like to keep it as `date: Date | string`

Reply:
The thing is: The signature component should not care about the type of string in the markdown file. They  should both not be related to each other. It's the job of `blog-template.tsx` to get the data and transform it in the correct format, so you can parse the date there and push it as normal date object to the signature.